### DISCUSSION TITLE
[Question][HW] About grace days

### DISCUSSION THREAD

**Discussion by cskwak99** [prosyslab-classroom/cs348-information-security#598] [Q&A]
Hello,
I have several questions about grace days. 

1. How can I check my remaining grace days?
2. If I want to use grace days, is it okay to just commit&push after deadline without any explicit work as in @24?

Thank you!

**Comment by RiceBiscuits** [comment]
1. Please send an email to TA.
2.  Yes. Grace day will be applied without any actions.

**Comment by cskwak99** [comment]
Thank you!
